<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><title>R: Information for features with an orf_on_chip present and...</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>

<table width="100%" summary="page for gene_info2_updated_20160903 {hiphop}"><tr><td>gene_info2_updated_20160903 {hiphop}</td><td style="text-align: right;">R Documentation</td></tr></table>

<h2>Information for features with an orf_on_chip present and mapped to current information from SGD as of 2016-09-03</h2>

<h3>Description</h3>

<p>No selection for strains to be excluded (see tag4_probe_info2 for that purpose )
</p>


<h3>Usage</h3>

<pre>
gene_info2_updated_20160903
</pre>


<h3>Format</h3>

<p>a data.frame with 6154 rows and 6 columns
</p>

<dl>
<dt>column names </dt><dd><p>self describing</p>
</dd>
<dt>gene_name </dt><dd><p>gene name from SGD but whenever there was no gene name, the associated sgd_orf was used</p>
</dd>
</dl>


<h3>Details</h3>

<p>This table helps just with mapping orf_on_chip with sgd_orf, so updated information can be obtained
</p>
<p>Information in part from: http://downloads.yeastgenome.org/curation/chromosomal_feature/SGD_features.tab
</p>
<p>However, whenever there was no gene_name, the corresponding sgd_orf was used in place
</p>

<hr /><div style="text-align: center;">[Package <em>hiphop</em> version 0.0.2 <a href="00Index.html">Index</a>]</div>
</body></html>
